web/docs/Backend Starters/Decoupled Drupal/configuring-build-hooks.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 'configuring-build-hooks'
+title: 'Configuring Build Hooks'
+slug: '/backend-starters/decoupled-drupal/configuring-build-hooks'
+sidebar_position: 5
+---
+
+## Setting Up Your First Build Hook
+
+### What Are Build Hooks?
+
+The Build Hooks module, available at https://www.drupal.org/project/build_hooks,
+allows CMS events like creating, updating, or deleting content to trigger builds
+in a decoupled frontend site.
+
+### Installing the Build Hooks Module
+
+To add Build Hooks as a composer dependency, run the following command:
+
+```
+composer require 'drupal/build_hooks:^3.3'
+```
+
```

Comment:
It might also be worth adding a note that by default the module will only log changes for content. If you'd like to trigger build hooks based on other entity updates,  Administration -> Configuration ->  Build hooks -> Build Hooks Settings.